### PR TITLE
The text reviewed.

### PR DESCRIPTION
**Pull request by alverena**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # Prerequirements
 
-[Gradle][1] is used as build tool for this project, so you have to install [it][1]. Of course jdk is also required.
+[Gradle][1] is used as a build tool for this project, so you have to install it. Of course, JDK is also required.
 
-## How to build plugin?
+## How to build the plug-in?
```
Comment (psliwa) [inline CONTRIBUTING.md]: I would prefer "plugin" rather than "plug-in". @iklymenko what do you think?

Comment (iklymenko) [inline CONTRIBUTING.md]: Yes, "plugin" is better.

Comment (iklymenko) [inline CONTRIBUTING.md]: @alverena could you please fix this.

**Comment by psliwa** [comment]
@alverena "IntelliJ SDK" is used correctly - it means "IntelliJ SKD" for plugin development: jars + sources with IntelliJ itself.

**Comment by iklymenko** [comment]
@alverena  Thanks for review!

**Pull request merged**